hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -1031,6 +1050,9 @@ public PropertyBuilder fromProperties(Properties properties) {
         setBootstrapIndexClass(
             hoodieConfig.getString(HoodieTableConfig.BOOTSTRAP_INDEX_CLASS_NAME));
       }
+      if (hoodieConfig.contains(HoodieTableConfig.BOOTSTRAP_INDEX_TYPE)) {
```

Comment:
I don't think this is correct? @codope

Reply:
good catch! This condition should call `setBootstrapIndexClass`.